kani-driver/src/args.rs

```diff
@@ -121,6 +121,9 @@ pub struct KaniArgs {
     /// Run Kani on all packages in the workspace.
     #[structopt(long)]
     pub workspace: bool,
+    /// Run Kani on the specified package.
+    #[structopt(long, short)]
+    pub package: Option<String>,
```

Comment:
I think this should be `Vec<String>` to allow specifying the flag multiple times (as `cargo build` does).

Reply:
422f2978c2e86343876997535067b39e5c824f01